templates/alert_form.html.twig

```diff
@@ -0,0 +1,252 @@
+    </div>
+
+    <div class="alert-preview">
+    </div>
+
+    <script>
+        (async function () {
+            const reload_preview = function () {
+                // Show preview section if hidden
+                $('.preview-section').removeClass('d-none');
+
+                // Send tinymce content to the forms textarea
+                tinyMCE.triggerSave();
+
+                // Load preview
+                $(".alert-preview").load("{{ preview_url }}" + "?" + $("form[name=asset_form]").serialize());
+            };
+
+            // Debounced function to not spam the backend for changes less than 0.4 seconds appart
+            const reload_preview_debounced = _.debounce(reload_preview, 400, false);
+
+            // Any input or dropdown change on our form
+            let updating_template = false;
+            $("form[name=asset_form] :input").on('input', function(e) {
+                if (updating_template) {
+                    // Prevent infinite loops as modifying the template will trigger
+                    // an update on others fields
+                    return;
+                }
+
+                if ($(e.target).prop('name') == 'type') {
+                    // Template change
+                    updating_template = true;
+
+                    // Template disabled, stop processing
+                    if ($(e.target).val() == "0") {
+                        updating_template = false;
+                        return;
+                    }
+
+                    // Clear values
+                    $("form[name=asset_form] :input[name=icon]").prop('checked', false);
+                    $("form[name=asset_form] :input[name=background_color]").prop('checked', false);
+                    $("form[name=asset_form] :input[name=text_color]").prop('checked', false);
+                    $("form[name=asset_form] :input[name=accent_color]").prop('checked', false);
+
+                    // Apply template values
+                    switch ($(e.target).val()) {
+                        {% for key, values in templates_values %}
+                            case '{{ key }}':
+                                $("form[name=asset_form] :input[name=icon][value={{ values.icon }}]").prop('checked', true);
+                                $("form[name=asset_form] :input[name=background_color][value={{ values.background_color }}]").prop('checked', true);
+                                $("form[name=asset_form] :input[name=text_color][value={{ values.text_color }}]").prop('checked', true);
+                                $("form[name=asset_form] :input[name=accent_color][value={{ values.accent_color }}]").prop('checked', true);
+                                break;
+                        {% endfor %}
+                    }
+
+                    updating_template = false;
+                } else if (['icon', 'background_color', 'text_color', 'accent_color'].includes($(e.target).prop('name'))){
```

Comment:
```suggestion
                } else if (['icon', 'background_color', 'text_color', 'accent_color'].includes($(e.target).prop('name'))) {
```

Reply:
Shouldn't it be caught by the lint ?
Done 037ab5f.